Feature/new home slider section

weberjavi opened the pull request:
This PR implements home redesign CW sections slider:
Slides designs could be found here:
https://projects.invisionapp.com/d/main#/console/16156416/335004751/preview

`IPSTACK_GEOLOCATION_API_KEY` env variable should be added to be able to get geolocator data.

aabdaab commented on app/javascript/app/styles/ui-extends.scss:
```diff
@@ -0,0 +1,42 @@
+  @extend %texts-defaults;
+
+  font-size: $font-size-xxl-large;
+  // line-height: $title-line-height;
```
remove comments pls 🙏

weberjavi replied:
🤦‍♂️

weberjavi commented:
@aabdaab deleted commented code and updated the case in which geolocation is not provided (disable the `Explore your country` button and changed dropdown placeholder text).

A change is needed on `Carousel` component to address the sliding issue.

weberjavi commented:
I was thinking about updating `cw-components` in another PR just to have this merged in the common branch to avoid conflicts. But it is alright with me to update component right away and change the version on this branch.
👍
